app/controllers/api/v2/hosts_controller.rb

```diff
@@ -38,6 +38,7 @@ def show
       def_param_group :host do
         param :host, Hash, :required => true, :action_aware => true do
           param :name, String, :required => true
+          param :aliases, String, :desc => N_("DNS aliases of the host.")
```

Comment:
Yeah, I like this approach, it would also make searching for host based on CNAME easier. But the interface could perhaps remain the same? Also I think it should not `belong_to :host` but rather to interface.